package.json

```diff
@@ -175,7 +175,7 @@
     "cozy-device-helper": "1.12.0",
     "cozy-doctypes": "1.82.2",
     "cozy-flags": "2.4.0",
-    "cozy-harvest-lib": "6.2.0",
+    "cozy-harvest-lib": "6.3.0",
```

Comment:
what more in this version?

Reply:
It uses deep imports and fixes the crash in Chrome while developing due to the pdf js worker.

Reply:
Thx